src/__integrationtests__/Attestation.spec.ts

```diff
@@ -68,23 +71,26 @@ describe('When there is an attester, claimer and ctype drivers license', () => {
     const claim = Claim.fromCTypeAndClaimContents(
       DriversLicense,
       content,
-      claimer.address
+      claimer.getAddress()
     )
-    const request = RequestForAttestation.fromClaimAndIdentity(
+    const [request] = await RequestForAttestation.fromClaimAndIdentity({
       claim,
-      claimer,
-      [],
-      null
-    )
+      identity: claimer,
+    })
     expect(request.verifyData()).toBeTruthy()
     expect(request.verifySignature()).toBeTruthy()
     const attestation = Attestation.fromRequestAndPublicIdentity(
       request,
-      attester.getPublicIdentity()
+      alice.getPublicIdentity()
     )
-    const result = await attestation.store(attester)
+    const result = await attestation.store(alice)
     expect(result.status.type).toBe('Finalized')
-    const aClaim = AttestedClaim.fromRequestAndAttestation(request, attestation)
+    const cred = await Credential.fromRequestAndAttestation(
+      claimer,
+      request,
+      attestation
+    )
+    const aClaim = cred.createPresentation([], false)
```

Comment:
See 559fafa for more explanation. As the first input refers to the properties which shall be excluded, no properties would be hidden in case of `[]` and all if I add all of them to the array. I believe this is quite clear, for each attribute you want to hide, you have to add it.

Reply:
Update: I renamed the first input variable of `cred.createPresentation` to `publicAttributes` in 08da080. Therefore, this reverses the explanation from above, e.g. if kept empty, all attributes will be hidden now. But, this does not interfere with `Claimer.createPresentation` where you insert attributes requested by the verifier, e.g. public attributes.